cpu/esp32/vendor/esp-idf/wpa_supplicant/port/Kconfig

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_ESP_IDF_WPA_SUPPLICANT_PORT
+    bool
+    depends on TEST_KCONFIG
+    depends on HAS_ARCH_ESP32
```

Comment:
```suggestion
```
This file was forgotten.

Reply:
Fixed 0caa902cddc17a52377716db5376a4a561e72eb7